### ISSUE TITLE
Page Templates - Test Home Template

### ISSUE THREAD

**Issue by vavalos5** [grommet/hpe-design-system#3123]
Use this [Figma Template
](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=5226%3A110552&t=mcK2Cf0fQBWkJqS0-1)to test with stakeholders.

## Problem Statement

As a designer, I am responsible for ideating common home pages that reflect use cases and needs. I need a home template with flexibility to quickly prototype a consistent HPE design system aligned design, confidently.

## Tasks

- [x] Create a test outline
- [x] Send to Kenny, Eric and Scott for review
- [x] Receive feedback from:
   - [x]  Kenny, 
   - [x] Eric 
   - [x] Scott
- [x] Implement feedback as needed
- [x] Set up calls with stakeholders
    - [x] Chris
    - [x]  Justin (confirmed on 2/6) [Figma Notes](https://www.figma.com/file/r8uxtnYgGyiS0yRQ1nYKiT/Page-Template-Study-Notes?node-id=9%3A94&t=7vyZBgueFbSTYfBM-0) | [Teams Recording](https://hpe-my.sharepoint.com/:v:/p/vicky_avalos/EZbsvSJnzwRDhUUYMFYuuPoBjmW-7E-pt1jJYuHNch7yzA)
    <s> - [x]  Gina (confirmed on 2/6)</s>
    - [x]  Amanda (confirmed on 2/6) [Figma Notes](https://www.figma.com/file/r8uxtnYgGyiS0yRQ1nYKiT/Page-Template-Study-Notes?node-id=1%3A46&t=hVhAQRADY8gDVn4k-1) | [Teams Recording](https://hpe-my.sharepoint.com/:v:/p/vicky_avalos/EW6j9dxVL2dCqu_a3dAejJ8BRfUUYWtZQpIve1Ro0tw40w)
   <s>  - [x] Megan (confirmed on 2/7)</s>
    <s> - [ ]  Aideen ( 8 hours ahead, might not be able to catch her)</s>
  <s>   - [ ]  Darren</s>
    <s> - [ ]  Dilip</s>
    - [x] Revanth (confirmed on 2/7)
   <s>  - [x]  Sherry (confirmed on 2/6) </s>
    - [x]  Edgar (confirmed on 2/6) [Figma Notes](https://www.figma.com/file/r8uxtnYgGyiS0yRQ1nYKiT/Page-Template-Study-Notes?node-id=9%3A71&t=7vyZBgueFbSTYfBM-0) | [Teams Recording](https://hpe-my.sharepoint.com/:v:/p/vicky_avalos/EY6NuKJbLUZLh9wkyid8GHYBaTkbXj9YEvJGPscjQPIcqg)
    - [x]  Ephrem (confirmed 2/3 and tested on 2/6) [Figma Notes](https://www.figma.com/file/r8uxtnYgGyiS0yRQ1nYKiT/Page-Template-Study-Notes?node-id=0%3A1&t=hVhAQRADY8gDVn4k-0) | [Teams Recording](https://hpe-my.sharepoint.com/:v:/p/vicky_avalos/Ed3FdkERPuFEo3Dm8lOXJYcBSvKKc39kIImHi44OXNn0ow)
   <s>  - [x]  Rena (confirmed on 2/6</s>
   <s> - [x]  Greg (confirmed on 2/6)</s>
- [x] Connect with stakeholders and test to understand whether the template created is easy or difficult to use.
- [x] Collect feedback
- [x] Finalize V1
- [ ] Receive feedback from:
   - [ ] Eric 
   - [x] Scott

## Acceptance Criteria

- [x] summary of test results added as a comment on this issue
- [ ] follow on issues created, especially noting where there are places Bill can evaluate examples for accessibility

## Success Criteria

Designer should be able to: 

- easily and quickly replicate template 

- easily and quickly edit content within template 

- utilize template to jumpstart their designs 

- feel that they have enough flexibility 

- feel confident utilizing template 

## Questions to consider

- What does a template mean to you?

- When thinking about a home page template, what is your expectation on what you'll find/see?

- Is having this template from XL-XS easy for you to understand how content behaves/stacks?

- Can you make a copy of this template and play around with it. Feel free to edit content within the cards, remove cards or add cards, or move things around to your liking. Please do also feel ok to talk out loud as you interact with the elements within this template. Once you are at a good place, let us know. 

- When using this template, how easy is it for you to add/remove/edit things?

- Is there anything that's missing?

- Any suggestions?

- Did this template meet your expectations when thinking about a home template?

** will be thinking about other questions

## Stakeholders

 Chris
 Justin
 Gina
 Amanda
 Megan
 Aideen
 Darren
 Dilip
 Revanth
 Sherry
 Edgar
 Ephrem
 Rena
 Greg

**Comment by vavalos5** [comment]
Adding @KennyAtHPE to this test to swarm together.

**Comment by vavalos5** [comment]
[Word document](https://hpe-my.sharepoint.com/personal/vicky_avalos_hpe_com/_layouts/15/doc.aspx?sourcedoc={b28d20a6-f9eb-4ed3-b124-e5cc4876f316}&action=edit) of where I've started a v1 outline of the test.

@KennyAtHPE @ericsoderberghp @SOjaHPE Can you please take a look and let me know if you see other opportunities or sections we don't want to include? Thanks!

**Comment by vavalos5** [comment]
Chatted w/ Scot this AM, here are some comments he made that we may want to add to the doc. I will be adding some later today and do a run through of how long i'm foreseeing this to take when testing w/ a user.

Do you really see a need to do any other template size? For instance, do you do this default for all of your designs? Do you really  Show them the different options, do you foresee 99% of your designs will be based off of this template?
 
Maybe talk to Derek and Greg on whether they have time to design for every size? How do you ensure your developer understands how things will behave when getting into smaller views? 

What would you expect as a template from DS? - Greg? 

Would you expect your template to drive the hierarchy of text? Semantically?  

Ask Chris if it would be benefit to create a content section component? 

Would you benefit from a content section? Do you expect there to be any other actions, buttons or anchors, etc..? PAL already supports this so why not eliminate it there and bring it to the DS.

**Comment by vavalos5** [comment]
I'd like to start setting calls tomorrow for next week. :)

**Comment by ericsoderberghp** [comment]
I read through the document. It looks good to me. I'm looking forward to seeing what we hear.

**Comment by KennyAtHPE** [comment]
@vavalos5 read through the document. Let's connect sometime tomorrow if you are available, I do have a couple of questions and am a bit confused in some areas. Maybe we can do a quick trial run?

**Comment by vavalos5** [comment]
- Walked Kenny through document.
- Next steps are to do a run through test with a user to get a better estimate on length.
- Based on that, we'll schedule the rest of the tests.
- Kenny will be starting a Word document to jot down notes
- I will be creating Figma files for each individual to play around with the template.

**Comment by vavalos5** [comment]
Kenny and I had our first test today with Ephrem. Notes are being taken within this[ Figma file](https://www.figma.com/file/r8uxtnYgGyiS0yRQ1nYKiT/Page-Template-Study-Notes?node-id=0%3A1&t=YW0apyz9CdQn308i-0).

[Figma file](https://www.figma.com/file/jwpZKPwCDlnD9EuWbfLwR8/Home-Page-Template-Testing?node-id=0%3A1&t=AWKPRvWqbnLRFjru-1) where users are interacting with template.

**Comment by vavalos5** [comment]
After chatting with Chris yesterday on 2/07, and Scott today on 2/08, we've decided that we're going to start polishing up the work i've already created and not focus on the study. We've cut down the list from the 11 people we were going to interview to 5: Amanda, Derek, Edgar, Revanth and Justin. We're just looking to get their perspective on whether this specific home page template is going to work for them when designing. From the studies already conducted, we feel confident in moving forward with what we've already created due to the positive feedback we've received including from Chris. 

The plan is to polish up whaat i've got by the end of the sprint.

**Comment by vavalos5** [comment]
I've cleaned up and completed V1 of the home page template and is ready for review. Please let me know your thoughts. @ericsoderberghp @SOjaHPE

**Comment by SOjaHPE** [comment]
@vavalos5  - thanks for the heads up. Looks great, I left 2 comments in your file.

**Comment by vavalos5** [comment]
We tested with Amanda on Tuesday. Today we tested with Justin and Edgar. Notes can be found in the Figma links placed above next to names.

**Comment by ericsoderberghp** [comment]
@vavalos5 - looks good to me, especially the "Templates in t-shirt sizing", something designers can copy/paste to start from.

**Comment by vavalos5** [comment]
[Figma file](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=5442%3A107658&t=ASWzQKjHQP4joKAm-1) with completed work
Two last things I added to my examples was the footer to the anatomy of the page layout since I hadn't added it. Also, after doing some testing with some folks, it came to our attention that people expect auto layout to be applied to the template in order to add and remove things without worrying about the responsiveness. So, I have added that as well.

Chris and I also chatted this AM about accounting for some bottom space per Matt's brand refresh work. While this doesn't affect us delivering this ticket, I will be adding a ticket in the backlog to explore. We can add those once we come to an agreement.

Since Scott and Eric have approved and i've walked Chris through this, I am closing.
